17style-elements/styles.css

```diff
@@ -0,0 +1,135 @@
+html,
+body {
+  height: 100%;
+}
+
+body {
+  display: flex;
+  margin: 0;
+  font: 100% / 1.5 sans-serif;
+  background: url("images/moon-bg.jpg") no-repeat fixed center / cover black;
+}
+.disclosure-alert {
```

Comment:
Typically there will be a line of whitespace between blocks to help make the code easier to read. Otherwise, the different selectors sort of run together.

```css
body {
  display: flex;
}

.another-selector {
  margin: 0;
}
```

Reply:
I just went through all the CSS and removed the spaces. It does look much better in terms of readability!